src/scss/components/_modal.scss

```diff
@@ -0,0 +1,152 @@
+:root {
+  // --transition: all 1s ease-in-out;
+  --transition: all 150ms ease-in-out;
+}
+.modal {
+  display: none;
+  @include breakpoint(md) {
+    display: block;
+    opacity: 0;
+    position: fixed;
+    overflow: hidden;
+    width: 100%;
+    height: 100%;
+    top: 0;
+    left: 0;
+    z-index: var(--z-index-modal);
+    pointer-events: none;
+    transition: var(--transition);
+
+    &__background {
+      position: absolute;
+      background: var(--brand-navy);
+      top: 0;
+      width: 100%;
+      height: 100%;
+      opacity: 0;
+      transition: var(--transition);
+    }
+    &__content {
+      position: fixed;
+      top: 50%;
+      left: 50%;
+      transform: translate(-50%, -70%);
+      display: flex;
+      width: 90%;
+      height: 90%;
+      width: 88rem;
+      height: 44.3rem;
+      border-radius: 2.4rem;
+      background: #fff;
+      opacity: 0;
+      transition: all 230ms ease-in-out;
+      position: relative;
+    }
+    &__image {
+      flex: 0 50%;
+      border-radius: 24px 0px 0px 24px;
+      overflow: hidden;
+      display: flex;
+      justify-content: center;
+      img {
+        width: fit-content;
+        max-width: fit-content;
+      }
+    }
+    &__description {
+      flex: 0 50%;
+      padding: 4.8rem 4rem;
+      h5,
```

Comment:
@annamarriap Can you add the class here and in the styles below?
for the h6, h5,p and a

Reply:
Sorry. I forgot, but is already done 34281fd620c7e20d5cada17be3fda4fac08ce473